style.css

```diff
@@ -0,0 +1,37 @@
+nav {
+    display: flex;
+    justify-content: flex-end;
+    position: fixed;
+    top: 0;
+    left: 0;
+}
+body {
+    margin: 0;
+}
+#welcome-section {
+    height: 100vh;
+}
+header {
+    position: fixed;
+    width: 100%;
+}
+nav a {
+    font-size: 10px;
+}
+
+@media only screen and (min-width: 400px) {
```

Comment:
You are actually doing a great job. I suggest you maintain consistency.  Leaving an empty line after each CSS rule can enhance readability for sure, but make it uniform for all the CSS rules. If you choose not to leave an empty line, let it be the same for all the rules.